CMakeLists.txt

```diff
@@ -284,6 +287,8 @@ mark_as_advanced(QUDA_OPENMP)
 
 mark_as_advanced(QUDA_BACKWARDS)
 
+mark_as_advanced(QUDA_QIOLARGEFILE)
```

Comment:
I think this would be best written as `QUDA_QIO_LARGE_FILE` to give better readability.